lib/game.rb

```diff
@@ -72,4 +78,43 @@ def diagonal_down_sort
     end
     padded_matrix.transpose.map {|array| array.compact}
   end
+
+  def game_menu
+    puts 'Welcome to Connect 4!! \n 
+      Enter p to Enter p to play the highest stakes game of your life. Enter q to wimp out and quit.'
+    ans = gets.chomp
+    if ans == "p"
+      @board = Board.new
+      turn_round
+    elsif ans == "q"
+      puts `clear`
+      puts 'GAME OVER'
+    else
+      puts `clear`
+      puts "#{ans} isn't a valid answer fool!!"
+      game_menu
+    end
+  end
+
+  def turn_round
+    until @game_win != nil
+      puts `clear`
+      puts "You chose #{@player_turns.last} and the computer chose #{@comp_turns.last}"
+      @board.render
+      puts "What column do you choose, intrepid player?"
+      player_turn(gets.chomp)
```

Comment:
Need to add some functionality, if a player chooses a full column or an invalid column then it should ask for a new selection.